hqqq

```diff
@@ -0,0 +1,76 @@
+diff --git a/util/hot_backup.cc b/util/hot_backup.cc
```

Comment:
now gone.  no clue how it was created.  but pretty clear on who let the random file into the repository.  me.  idiot.